### PR TITLE
Save wheels as test job artifacts

### PR DESCRIPTION
**Pull request by malfet**

### CODE REVIEW

File: .circleci/unittest/linux/scripts/install.sh [hunk 1]
```diff
@@ -23,20 +23,24 @@ conda activate ./env
...
 git submodule sync
 git submodule update --init --recursive
-conda install -y numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses hypothesis
+conda install -y numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses hypothesis wheel
+
 if [ "${CU_VERSION:-}" == cpu ] ; then
     printf "* Installing NT-specific pytorch and nestedtensor cpu-only\n"
     pushd third_party/pytorch
-    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=OFF BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop
+    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=OFF BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -b $WHEELS_FOLDER
```
Comment (malfet) [inline .circleci/unittest/linux/scripts/install.sh:37]:
```suggestion
    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=OFF BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -d $WHEELS_FOLDER
```

File: .circleci/unittest/linux/scripts/install.sh [hunk 2]
```diff
@@ -23,20 +23,24 @@ conda activate ./env
...
     popd
     USE_NINJA=1 python setup.py develop
```
Comment (malfet) [inline .circleci/unittest/linux/scripts/install.sh:39]:
```suggestion
    USE_NINJA=1 python setup.py develop bdist_wheel -d $WHEELS_FOLDER
```

File: .circleci/unittest/linux/scripts/install.sh [hunk 3]
```diff
@@ -23,20 +23,24 @@ conda activate ./env
...
 else
     printf "* Installing NT-specific pytorch and nestedtensor with cuda\n"
     pushd third_party/pytorch
-    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=ON  BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop
+    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=ON  BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -b $WHEELS_FOLDER
```
Comment (malfet) [inline .circleci/unittest/linux/scripts/install.sh:43]:
```suggestion
    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=ON  BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -d $WHEELS_FOLDER
```

File: .circleci/unittest/linux/scripts/install.sh [hunk 4]
```diff
@@ -46,5 +50,5 @@ rm -rf /tmp/vision
 git clone https://github.com/pytorch/vision /tmp/vision
 
 pushd /tmp/vision
-python setup.py develop
+python setup.py develop bdist_wheel -b $WHEELS_FOLDER
```
Comment (malfet) [inline .circleci/unittest/linux/scripts/install.sh:53]:
```suggestion
python setup.py develop bdist_wheel -d $WHEELS_FOLDER
```

File: .circleci/unittest/linux/scripts/install.sh [hunk 5]
```diff
@@ -23,20 +23,24 @@ conda activate ./env
...
+    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=OFF BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -b $WHEELS_FOLDER
     popd
     USE_NINJA=1 python setup.py develop
 else
     printf "* Installing NT-specific pytorch and nestedtensor with cuda\n"
     pushd third_party/pytorch
-    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=ON  BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop
+    USE_DISTRIBUTED=ON BUILD_TEST=OFF USE_CUDA=ON  BUILD_CAFFE2_OPS=0 USE_NUMPY=ON USE_NINJA=1 python setup.py develop bdist_wheel -b $WHEELS_FOLDER
     popd
     USE_NINJA=1 python setup.py develop
```
Comment (malfet) [inline .circleci/unittest/linux/scripts/install.sh:45]:
```suggestion
    USE_NINJA=1 python setup.py develop bdist_wheel -d $WHEELS_FOLDER
```

**Comment by cpuhrsch** [comment]
Installed locally via
```
pip install https://3278-217161669-gh.circle-artifacts.com/0/wheels/torch-1.7.0a0-cp37-cp37m-linux_x86_64.whl https://3278-217161669-gh.circle-artifacts.com/0/wheels/nestedtensor-0.0.1.dev202011320-cp37-cp37m-linux_x86_64.whl https://3278-217161669-gh.circle-artifacts.com/0/wheels/torchvision-0.9.0a0%2B5b61a5c-cp37-cp37m-linux_x86_64.whl
```
and tests pass

**Pull request merged**